packages/dashboard-core-plugins/src/panels/ConsolePanel.tsx

```diff
@@ -368,6 +375,11 @@ export class ConsolePanel extends PureComponent<
       timeZone,
       unzip,
     } = this.props;
+
+    if (sessionWrapper == null) {
+      return null;
```

Comment:
We shouldn't display _nothing_ I don't think we should be displaying an error message. When do we expect this to occur?

Reply:
This happens when logging out and in some cases when logging in

Reply:
It can also happen if remote consoles are disabled. You can do that by running with the flag `deephaven.console.disable=true`, e.g.:
```
START_OPTS="-Ddeephaven.console.disable=true" ./gradlew server-jetty-app:run -Ppsk=iris
```
In this case we're displaying a blank console panel now:
<img width="1058" alt="image" src="https://github.com/deephaven/web-client-ui/assets/4505624/7e2c3151-8829-46c3-96e7-2a9869f048f6">

Ideally in that scenario, we should display an error message. So I think we should display an appropriate error message here. It is still an edge case though; by default, if remote console is enabled, we won't even add a console panel to the layout. But if they already have a layout saved locally with the ConsolePanel, or they import a layout with ConsolePanel, they'll run into this error.

On Logout we immediately transition back to the login screen, so no concern about the error message appearing when logging out.

Reply:
@dsmmcken any suggestions on error message here?

Reply:
Is this two seperate error states?

1. "Console option is disabled."
2. "Unable to load console, no available session."

Reply:
Yes, although I'm not sure the user will ever actually see #2

Reply:
Is there a standard icon for this?

Reply:
Use a `LoadingOverlay` with `isLoading={null}` and the error message set, e.g.
```
      <LoadingOverlay
        isLoading={false}
        errorMessage="Console option is disabled."
      />
```